docs/ERC20.md

```diff
@@ -124,13 +127,47 @@ await signer.send_transaction(account, erc20.contract_address, 'transfer', [reci
 
 ## Extensibility
 
-There's no clear contract extensibility pattern for Cairo smart contracts yet. In the meantime the best way to extend our contracts is copypasting and modifying them at your own risk.
+ERC20 contracts can be extended by following the [extensibility pattern](../docs/Extensibility.md#the-pattern). The basic idea behind integrating the pattern is to import the requisite ERC20 methods from the ERC20 library and incorporate the extended logic thereafter. For example, let's say you wanted to implement a pausing mechanism. The contract should first import the ERC20 methods (and the extended logic if it's in a separate contract). Next, the contract should expose the methods with the extended logic therein like this:
```

Comment:
> (and the extended logic if it's in a separate contract)

I didn't get that part, what do you mean?

Reply:
Ah I meant library, not contract. I was trying to convey an example like how `ERC20_Pausable` combines functions from two libraries (`ERC20` and `pausable`) into the exposed functions—as opposed to importing from just one library and having the extended behavior coded in the contract itself. I don't think I phrased this well :) I'll edit

Reply:
Definitely! I would try to convey that extensibility can be custom as well, not just library-based.